src/TesApi.Web/BatchScheduler.cs

```diff
@@ -169,8 +154,7 @@ public partial class BatchScheduler : IBatchScheduler
                 _batchPoolFactory = poolFactory;
                 batchPrefix = batchSchedulingOptions.Value.Prefix;
                 logger.LogInformation("BatchPrefix: {BatchPrefix}", batchPrefix);
-                taskRunScriptContent = File.ReadAllLines(Path.Combine(AppContext.BaseDirectory, "scripts/task-run.sh"));
-                taskCleanupScriptContent = File.ReadAllLines(Path.Combine(AppContext.BaseDirectory, "scripts/clean-executor.sh"));
+                File.ReadAllLines(Path.Combine(AppContext.BaseDirectory, "scripts/task-run.sh"));
```

Comment:
If this isn't storing the contents of the file anywhere, this line can go away. If that is the case, please also delete the two files in the `scripts` directory.